packages/core-utils/src/type-utils/LoadedState.ts

```diff
@@ -18,6 +18,4 @@ export interface ErrorState<S, E = Error> {
   error: E;
 }
 
-type LoadedState<S, E = Error> = NoneState | LoadingState<S> | ReadyState<S> | ErrorState<S, E>;
-
-export default LoadedState;
+export type LoadedState<S, E = Error> = NoneState | LoadingState<S> | ReadyState<S> | ErrorState<S, E>;
```

Comment:
Since all imports will be from the package (`import { LoadedState } from '@terra-ui-packages/core-utils';`), all exports must be named instead of default.